src/Keboola/ConfigMigrationTool/Service/OAuthService.php

```diff
@@ -12,10 +12,10 @@ class OAuthService
     /** @var Client */
     private $client;
 
-    public function __construct()
+    public function __construct(?string $baseUrl = 'https://syrup.keboola.com/oauth-v2/')
```

Comment:
Jakej je rozdíl mezi https://github.com/keboola/config-migration-tool/blob/master/src/Keboola/ConfigMigrationTool/Service/OAuthService.php a https://github.com/keboola/config-migration-tool/blob/master/src/Keboola/ConfigMigrationTool/Test/OAuthV2Service.php ?
Náhodou jsem na to teď narazil.

Reply:
No, v podstate ziadny, to ma tak trochu donutil spravit Odin :D 
https://github.com/keboola/config-migration-tool/pull/28

Ale ja som myslel, ze to bude pouzite len v testoch, takze to bolo povodne len v Tests namespace, ale teraz som si vsimol, ze to pouzivam aj v "normalnom" kode, takze som tuto triedu prehodil do "normalneho" namespace. 

Vyriesim to v dalsom PR.

Reply:
no protoze jsi predtim upravoval kod jen kvuli tomu, ze jsi delal neco v testech. Podle me by bylo nejlepsi dat https://github.com/keboola/config-migration-tool/blob/master/src/Keboola/ConfigMigrationTool/Test/OAuthV2Service.php uplne cely dohaje a namockovat to a bude klid.

Reply:
jo ted to vidím že si to v rámci tohodle PR přesunul. Nešlo by ty úpravy souvsiející s migrací Oauth udělat až v dalším PR? tohle mělo jenom zprovoznit multi stack podporu pro docker migrace

Reply:
kolem toho oauthu mi přijde že je tam víc problémů které bych řešil separátně. Např. tady používáš nějakou `env` v kódu https://github.com/keboola/config-migration-tool/pull/37/files#diff-efe42b97daa884ef43cd3b142a4a783dR29 .

Reply:
ty migrace oauth ještě nejsou publikovaný/připravený jestli se nepletu?

Reply:
Este to nie je hotove, ta `env` je tam v podstate kvoli testom, to by sa asi malo spravit inak veru.  OK. Mozem to z tohto PR vyhodit -> vratit tu triedu do "Test" namespace. 
A potom to budem riesit dalej. Mozno to v tych testoch pre OAuth naozaj namockujem ako pise Odin.

Reply:
jo vyhod prosim ty zmeny kolem oauth tady, to se dotahne separatne. 
A muze se to pak mergnout.

Reply:
OK, revertol som to. Aj ked viac-menej rucne, lebo cherry-picking by bol v tom druhom pripade na dlho.